### PR TITLE
DRFT-661 - Add pytest plugins

### PR DESCRIPTION
**Pull request by fellipeh**
This PR:
- Add some pytest plugins
- make all config into a single file: pyproject.toml

Signed-off-by: Fellipe Henrique <fpedrosa@redhat.com>

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: run_unit_tests.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 TEMPDIR=`mktemp -d`
 
-prometheus_multiproc_dir=$TEMPDIR pytest ./tests --cov-report term-missing --cov=. --cov-fail-under=73  --no-cov-on-fail "$@" && prometheus_multiproc_dir=$TEMPDIR python generate_report.py test_reports.toml && rm -rf $TEMPDIR
+prometheus_multiproc_dir=$TEMPDIR pytest  "$@" && prometheus_multiproc_dir=$TEMPDIR python generate_report.py test_reports.toml && rm -rf $TEMPDIR
```
Comment (CodeHeeler) [inline run_unit_tests.sh:8]: Is there a way to still include coverage reports/percentages here like before?

**Review by CodeHeeler — APPROVED** [review]
Added a non-blocking question.

LGTM, thank you for all this pytest work!

File: run_unit_tests.sh (hunk 1 above)
Comment (fellipeh) [reply run_unit_tests.sh:8]: Now all this config is inside the `pyproject.toml`

Comment (petrblaho) [reply run_unit_tests.sh:8]: now I see it too :-)
it is at https://github.com/RedHatInsights/drift-backend/pull/272/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R26

**Review by petrblaho — APPROVED** [review]

**Comment by fellipeh** [comment]
I'll merge since travis ci passed: https://travis-ci.com/github/RedHatInsights/drift-backend/builds/232622185
just not tell github this..

**Pull request merged**